src/lib.rs

```diff
@@ -2041,8 +2044,11 @@ impl Connection {
     /// # let scid = quiche::ConnectionId::from_ref(&[0xba; 16]);
     /// # let mut conn = quiche::accept(&scid, None, &mut config)?;
     /// loop {
-    ///     let write = match conn.send(&mut out) {
-    ///         Ok(v) => v,
+    ///     let write = match conn.send_with_info(&mut out) {
+    ///         Ok((v, send_info)) => {
+    ///             // Use send_info
+    ///             v
+    ///         },
```

Comment:
```suggestion
    ///     let (written, send_info) = match conn.send_with_info(&mut out) {
    ///         Ok(v) => {
    ///             v
    ///         },
```

As far as examples go, it would probably be clearer here to just return the 2-tuple here and then later, before the socket send, give an example of using send_info. Which actually identifies to me a bit of confusion - what exactly am I supposed to do with SendInfo.send_time? Maybe we should update the examples?